socorro/unittest/lib/test_javautil.py

```diff
@@ -100,8 +100,8 @@ def test_parse_suppressed():
     assert java_exc.exception_message == "msg"
     assert java_exc.stack == ["at org.File.function(File.java:100)"]
     assert java_exc.additional == [
-        "Suppressed: Exception2: msg2",
-        "\tat org.File.function(File.java:101)",
+        "\tSuppressed: Exception2: msg2",
+        "\t\tat org.File.function(File.java:101)",
```

Comment:
Because we took out the "let's remove the first tab from additional bits", we needed to update the existing tests.